robottelo/ui/user.py

```diff
@@ -156,12 +159,14 @@ def update(self, username, new_username=None,
             default_loc=default_loc,
             select=select
         )
-        self.click(common_locators['submit'])
-
-    def admin_role_to_user(self, username):
-        """Checks if selected user has Administrator privileges otherwise
-        assign it to user.
+        if submit:
+            self.click(common_locators['submit'])
+        else:
+            self.click(common_locators['cancel_form'])
 
+    def admin_role_to_user(self, username, assign=True):
```

Comment:
also rename the input attribute to this function i.e., `s/assign=True/is_admin=True`